src/Control Tasks/CameraControlTask.cpp

```diff
@@ -35,6 +35,7 @@ void CameraControlTask::execute()
             transition_to_normal();
         }
         if (sfr::camera::init_mode == camera_init_mode_type::failed) {
+            if(sfr::camera::failed_times < 10) sfr::camera::failed_times = sfr::camera::failed_times + 1;
```

Comment:
I believe we are using a common format - not sure if it disallows unscoped if statements

Reply:
I add the `if(sfr::camera::failed_times < 10)` here for avoiding overflow. But I am not sure if this is necessary, I will make sure with Lauren in the meeting

Reply:
Is sfr::camera::failed_times ever reset? This would resolve the sensor overflow issue and allow us to enter this mode multiple times.